components/dropdown.scss

```diff
@@ -1,45 +1,47 @@
-$dropdown-border-radius: $border-radius;
-$dropdown-border: 1px solid var(--hdd-color-grey);
+$dropdown-border: 1px solid var(--hdd-color-blue);
 $dropdown-margin: 20px;
-$dropdown-spacing-element: 4px;
-$dropdown-content-padding: 0.5rem 0;
-$dropdown-item-padding: 0.5rem 0 0.5rem 1rem;
+$dropdown-item-padding: 0.625rem 0.75rem;
 
 .hdd-dropdown {
   position: relative;
   box-sizing: border-box;
 
   &_content {
     @extend .is-aligned-bottom;
-
-    display: none;
+    display: none;    
     position: absolute;
-
     z-index: 90;
-    left: 0;
     min-width: 100%;
-
-    padding: $dropdown-content-padding;
+    max-height: 150px;
+    overflow-y: auto;
+    // TODO: removes white line from top of dropdown, find better solution. 
+    margin-top: 1px;
```

Comment:
Får ikke til å gjenskape white line problemet når jeg fjerner margin-top: 1px

Reply:
Merkelig, har prøvd selv nå og det fungerer fint, reverterte tilbake til gammel løsning med bare 2px på den is-aligned bottom attributten og nå fungerer det fint.